src/app/login/page.tsx

```
@@ -0,0 +1,97 @@
+              <span className="w-full border-t" />
+            </div>
+            <div className="relative flex justify-center text-xs uppercase">
+              <span className="bg-white px-2 text-gray-500">or</span>
+            </div>
+          </div>
+        </div>
+
+        {/* Login Form */}
+        <form className="space-y-4">
+          <div className="space-y-2">
+            <label className="text-sm text-gray-600">Email Address</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)